cpp/velox/memory/VeloxMemoryPool.cc

```diff
@@ -373,33 +396,36 @@ class WrappedVeloxMemoryPool final : public velox::memory::MemoryPool {
     visitor(subtreeMemoryUsage_);
   }
 
-  int64_t cap_;
-  velox::memory::MemoryManager& memoryManager_;
+  const std::shared_ptr<velox::memory::MemoryUsageTracker> memoryUsageTracker_;
+  velox::memory::MemoryManager* const memoryManager_;
+  velox::memory::MemoryAllocator* const allocator_;
+  const DestructionCallback destructionCb_;
 
   // Memory allocated attributed to the memory node.
   velox::memory::MemoryUsage localMemoryUsage_;
-  std::shared_ptr<velox::memory::MemoryUsageTracker> memoryUsageTracker_;
   mutable folly::SharedMutex subtreeUsageMutex_;
   velox::memory::MemoryUsage subtreeMemoryUsage_;
-  velox::memory::MemoryAllocator& allocator_;
   std::shared_ptr<VeloxMemoryAllocatorVariant> glutenAllocator_;
 };
 
 std::shared_ptr<velox::memory::MemoryPool> AsWrappedVeloxMemoryPool(gluten::MemoryAllocator* allocator) {
   return std::make_shared<WrappedVeloxMemoryPool>(
-      velox::memory::MemoryManager::getInstance(),
-      "wrapped",
+      &velox::memory::MemoryManager::getInstance(),
+      "wrappedRoot",
+      velox::memory::MemoryPool::Kind::kAggregate,
```

Comment:
Why is `kAggregate` used here?

Reply:
As velox introduces "kAggregate, kLeaf",[link](https://github.com/zhejiangxiaomai/velox/blob/main_rebase_327/velox/common/memory/MemoryPool.h#L91-L99) I think AsWrappedVeloxMemoryPool should return a "kAggregate" type memory pool which can use addChild to create new memory pool for caller.